openedx/features/partners/models.py

```diff
@@ -7,6 +7,7 @@ class Partner(TimeStampedModel):
     """
     This model represents white-labelled partners.
     """
+    performance_url = models.URLField(blank=True, default=None)
```

Comment:
@ahmed-faraz46 have you tested it with null value ?

Reply:
Yes